app/Models/Employee.php

```diff
@@ -16,5 +19,21 @@ class Employee extends Model
         'dob',
         'phone',
         'address',
+        'company_id'
     ];
+
+    public function company():BelongsTo
+    {
+        return $this->belongsTo(Company::class);
+    }
+
+    public function projects(): BelongsToMany
+    {
+        return $this->belongsToMany(Project::class);
+    }
+
+    public function phone(): HasOne
```

Comment:
fix the relationship naming

Reply:
relation naming fixed #28  @Raza9798  please review again

Reply:
have you push the changes? i dont see any changes @axnish27

Reply:
sorry seems like i ddnt...Now pushed please check again